Makefile

```diff
@@ -1,6 +1,6 @@
 ADB = ${ANDROID_HOME}/platform-tools/adb
 GRADLEW = ./gradlew
-flavour = Unbranded
+flavour = cmmb_kenya
```

Comment:
i think this is making the default falvour cmmb?

Reply:
Nice one. Forgot to revert that